app/controllers/api/floating_ips_controller.rb

```diff
@@ -1,4 +1,40 @@
 module Api
   class FloatingIpsController < BaseController
+    def create_resource(_type, _id = nil, data = {})
+      ems = ExtManagementSystem.find(data['ems_id'])
+      klass = FloatingIp.class_by_ems(ems)
+      raise BadRequestError, "Create floating ip for Provider #{ems.name}: #{klass.unsupported_reason(:create)}" unless klass.supports?(:create)
+
+      task_id = ems.create_floating_ip_queue(session[:userid], data.deep_symbolize_keys)
+      action_result(true, "Creating Floating Ip #{data['name']} for Provider: #{ems.name}", :task_id => task_id)
+    rescue => err
+      action_result(false, err.to_s)
+    end
+
+    def edit_resource(type, id, data)
+      floating_ip = resource_search(id, type, collection_class(:floating_ips))
+      raise BadRequestError, "Update for #{floating_ip_ident(floating_ip)}: #{floating_ip.unsupported_reason(:update)}" unless floating_ip.supports?(:update)
```

Comment:
I just noticed that the supports feature is `:update` but the unsupported reason is `:update_floating_ip` [[ref]](https://github.com/ManageIQ/manageiq-providers-openstack/blob/master/app/models/manageiq/providers/openstack/network_manager/floating_ip.rb#L18) so you likely won't see the proper error here but that looks like a bug on the openstack side not here